cube/src/main/java/zone/cogni/asquare/cube/convertor/ModelToJsonConversion.java

```diff
@@ -214,7 +214,7 @@ private void processContext(Context context, Model model) {
   }
 
   private void reportMissedSubjects(Context context, String root) {
-    Set<Resource> missedSubjects = context.subjectTypeMap.keySet();
+    Set<Resource> missedSubjects = new HashSet<>(context.subjectTypeMap.keySet());
```

Comment:
note: fixes a "counting" bug. this is only logged if reporting of missing data is enabled!